components/sections/Tabs/Tabs.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../../styles/fonts.scss";
+
+.tabs {
+  min-height: 300px;
+  padding-top: 50px;
+  padding-bottom: 50px;
+  font-family: $poppins-semibold;
+}
+
+.tabsName {
```

Comment:
Use underscores. They will be converted to camel case automagically.

Reply:
I wouldn't have a way of writing classes names differently from the way we access them. We could decide if we want everything with underscore or with camel case.

Reply:
Sorry, I meant dashes. CSS convention is to use dashes. Javascript convention is to use camel case. I prefer to keep each language convention, and the framework offers a tool for it, which means that conversion from dashes to camel case is a convention imposed by the framework as well.

Reply:
CSS doesn't have any conventions on how to write class names. In the FD UI we used camel case for every class.
It is true that CSS doesn't have camel case per se, as it's case insensitive, but camel casing is only used by us to read it properly.